README.md

```diff
@@ -13,47 +13,130 @@ Before proceeding, make sure you have installed:
 
```

Comment:
Can you add git to Prerequisites?

Reply:
lemme think about it and ill get back to you